skyportal/handlers/api/spectrum.py

```diff
@@ -195,6 +196,14 @@ def get(self, spectrum_id):
             .filter(CommentOnSpectrum.spectrum_id == spectrum_id)
             .all()
         )
+        annotations = (
+            AnnotationOnSpectrum.query_records_accessible_by(
+                self.current_user,
+                options=[joinedload(AnnotationOnSpectrum.groups)],
```

Comment:
Does the front-end even utilize the groups list for each annotation? If not, let's omit this

Reply:
Maybe you are right...